wafo-server/src/main/java/wafo/wafoserver/link/service/LinkService.java

```diff
@@ -0,0 +1,47 @@
+package wafo.wafoserver.link.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import wafo.wafoserver.account.user.User;
+import wafo.wafoserver.account.user.repository.UserRepository;
+import wafo.wafoserver.folder.Folder;
+import wafo.wafoserver.folder.repository.FolderRepository;
+import wafo.wafoserver.link.Link;
+import wafo.wafoserver.link.controller.dto.LinkCreateRequest;
+import wafo.wafoserver.link.controller.dto.LinkCreateResponse;
+import wafo.wafoserver.link.dto.CreateLinkDto;
+import wafo.wafoserver.link.repository.LinkRepository;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class LinkService {
+    private final LinkRepository linkRepository;
+    private final UserRepository userRepository;
+    private final FolderRepository folderRepository;
+
+    @Transactional
+    public LinkCreateResponse create(LinkCreateRequest linkCreateRequest) {
+
+        User user = userRepository.findById(linkCreateRequest.getUserId()).get();
+        Folder folder = folderRepository.findById(linkCreateRequest.getFolderId()).get();
```

Comment:
이 부분에 디폴트 폴더가 들어가는 거라면

LinkCreateRequest에 상수값을 넣어주거나 
아래처럼 상수값을 사용해도 좋을 것 같습니다 

그럴려면 InitDB에 디폴트 폴더를 만들어줘야할 거 같습니다.

```suggestion
Folder folder = folderRepository.findById(DEFAULT_FOLDER_ID).get();
```

Reply:
현재 InitDB에 디폴트 폴더는 만들어 놓았습니다. *이전 커밋 참조

현재 로직은 경우의 수가 두 가지라고 생각합니다.
1. 사용자가 폴더를 미지정한 경우
2. 사용자가 폴더를 지정한 경우
이에 대한 공통적인 처리를 위해 위와 같이 작성하였습니다.

다만, 현재는 무조건 디폴트 폴더에 저장하는 형식으로 작성 후
추후 LinkCreateRequest 등에서 사용자가 저장하려는 폴더를 지정 or 미지정했는지 판단 후
올바른 폴더에 Link가 저장될 수 있도록 수정하겠습니다.